src/sqlfluff/dialects/dialect_snowflake.py

```diff
@@ -1570,16 +1563,6 @@ class AlterWarehouseStatementSegment(BaseSegment):
     )
 
 
-class CommentClauseSegment(BaseSegment):
```

Comment:
This entire segment definition is identical to the base ANSI dialect, so I removed it.